_data/donation-software.yml

```diff
@@ -61,6 +61,13 @@ websites:
       othercrypto: Yes
       doc: https://pi-hole.net/donate/
 
+    - name: PyPI
```

Comment:
I think it is better to use Python Software Foundation as the donation name, PyPI is one of the services they are providing.